runfolder/services.py

```diff
@@ -153,13 +161,19 @@ def get_runfolder_state(self, runfolder):
         If the file .arteria/state exists, it will determine the state. If it doesn't
         exist, the existence of the marker file RTAComplete.txt determines the state.
         """
-        completed_marker_file = self._configuration_svc["completed_marker_file"]
+        completed_marker_files = self._configuration_svc["completed_marker_files"]
```

Comment:
I think it would be good to ensure that the returned value is a list and not a string, otherwise we would see some unexpected and potentially hard-to-debug behaviour caused by mistakes in the configuration.. especially since the previous version expected a string.